content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -141,9 +141,10 @@ validation error during StatefulSet creation.
 
 ### Volume Claim Templates
 
-You can set the `.spec.volumeClaimTemplates` which can provide stable storage using
+You can set the `.spec.volumeClaimTemplates` field to provide stable storage using
 [PersistentVolumes](/docs/concepts/storage/persistent-volumes/) provisioned by a PersistentVolume
-Provisioner.
+Provisioner. Make sure that a PersistentVolume with matching `storage class` 
+and sufficient available storage space already exists in the cluster. 
```

Comment:
You don't need to make sure that the PV already exists; instead, you can set up your cluster so that claiming storage triggers automatic provisioning.

Reply:
That's great to know, thanks. I think it would help to explain that clearly on this page so I'll read up a bit on volume provisioning and then update the PR.

Reply:
Hi Tim,

I just pushed an update. What do you think? I think it's important to emphasize that the volume claim template field only creates a PVC and that you need to make sure it can be bound to an actual PV.

Reply:
Seems OK.